apps/showcase/src/app/sdk-training/sdk-training.component.scss

```diff
@@ -0,0 +1,3 @@
+o3r-training {
+  height: calc(100vh - 4rem);
```

Comment:
why `4rem` ?

Reply:
We can add a comment, 4rem is the size of the header, it is to ensure the component fits in the screen :) 
(vh -> visible height on the screen)